features/bootstrap/RabbitMqContext.php

```diff
@@ -108,7 +108,11 @@ function (bool $carry, AMQPMessage $message) use ($expectedHash): bool {
             );
 
             if (!$found) {
-                throw new \LogicException('Expected message not found');
+                $strMessages = '';
```

Comment:
Messages looks glued without space using that code 🤔, though unreadable. I would do it like this:
```php
$messagesString = implode("\n", array_map(function ($msg) { return $msg->getBody(); }, $messages));

throw new \LogicException("Expected message not found among these ones:\n$messagesString");
```
WDYT?